packages/typedoc-plugins/package.json

```diff
@@ -22,8 +22,8 @@
   ],
   "devDependencies": {
     "chai": "^4.2.0",
-    "glob": "^10.0.0",
-    "mocha": "^10.0.0",
```

Comment:
The `mocha` bump to v11.x prevents execution of `yarn run test` script on Windows for unknown reasons (probably due to internal bump of `yargs` to v17.x in `mocha`).

```
Error: No test files found: "'./tests/**/*.js'"
```

It starts to work again after replacing `'` with `"`:

```
"mocha \"./tests/**/*.js\" --timeout 10000"
```

Reply:
Done :heavy_check_mark: